modules/oe_webtools_geocoding/config/schema/oe_webtools_geocoding.schema.yml

```diff
@@ -1,3 +1,9 @@
 geocoder_provider.configuration.webtools_geocoding:
   type: geocoder_provider_configuration
   label: 'Webtools Geocoding configuration'
+  mapping:
+    referer:
+      type: string
+      label: 'Referer'
+      description: 'Value of the Referer header'
+      nullable: true
```

Comment:
This shows that my comment here https://github.com/openeuropa/webtools-geocoding-provider/pull/6#discussion_r328519564 was correct.